### ISSUE TITLE
make binge mode optional

### ISSUE THREAD

**Issue by justchokingaround** [justchokingaround/jerry#76]

**Comment by rwmpelstilzchen** [comment]
Yeah! Having to wait for the next episode to load only to quit immediately after it starts makes no sense. In fact, IMHO binge mode shouldn’t even be the default when it’s optional.

Thx for the great script 🌼
